docs/scanner-pools.md

```diff
@@ -0,0 +1,158 @@
+
+- Navigate to “My Stakes” or  “Overview” on your profile and you can see your staked FORT
+
+## Creating new nodes
+
+Each node needs to be registered to a specific pool in order to be operational. While one can register many scan nodes on a pool, the pool needs to be populated with more stake first in order to avoid registration issues. This is for preventing pool shutdown conditions that occur when the average per-scanner stake is below minimum.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to add a node
+![My Node Pools List](MyNodePoolsList.png)
+
+- Make sure you have at least 2,500 FORT staked for the new node. Click “Add Scan Node”. Use the Forta node CLI to generate the scanner authorization token (described [here](scanner-quickstart#register-scan-node)) needed to claim a node into a pool.
+
+![Add Scan Node](AddScanNode.png)
+
+## Disabling existing nodes
+
+If a node is no longer used, it should be disabled to free up allocated stake for newer nodes.
+
+- Navigate to the node pool that has the node you want to disable
+
+- Select the disable action on the node you want to disable
+
+![Disable Scan Node](DisableScanNode.png)
+## Adjusting stake allocation
+
+While it's always possible to add more stake to a pool, it is allocated proportional to the amount of scan nodes in the pool. Total allocated stake can only be `node_count × max_stake_per_scan_node`.
+
+For example, if there are two scan nodes in a pool, a total of 25k FORT deposited and only 5k FORT worth of stake is allowed per scan node, then the pool can allocate a maximum of `5k × 2 = 10k` FORT in total. This is the amount that is effective in calculating the rewards and the rest of `25k - 10k = 15k` remains unallocated until more scan nodes are registered to the pool.
+
+It is up to the pool owner to allocate more from own stake or more from the delegators so the share of the rewards can be adjusted accordingly.
+
+### Adjusting stake
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to adjust the stake allocation
+
+![My Node Pools List](MyNodePoolsList.png)
+
+- Click on "Allocate Stake"
+
+![Allocate Stake On Node](AllocateStakeOnNode.png)
+
+- From here you can either allocate or unallocate stake. To adjust you own stake make sure the subject is *Owner*
+
+![Allocate Modal](AllocationModal.png)
+
+- To adjust the stake of delegators select the *Delegators* subject
+
+![AllocateDelegates](AllocateDelegates.png)
+## Changing the commission
+
+The pool owners can adjust the ratio of the shares they would like to keep for themselves or the delegators, by changing the allocated amounts as described in the previous sections. It can be the case that it's more desirable for the operator to not stake more on the pool but let the delegators fill up the available allocation capacity.
```

Comment:
an example might help clarify how commission works

Reply:
also, probably worth mentioning a few important points about commission:
- default commission for a pool is 100% (i.e. owner keeps all rewards). this can be used to disincentivize others from delegating on your pool if you wish to do so
- commission changes dont take effect until the next epoch (which begins every Monday 00:00 GMT)

Reply:
Added an example and mentioned the 100%.

> commission changes dont take effect until the next epoch (which begins every Monday 00:00 GMT)

I think it would be strange to say "next epoch". By the time we make the release, the operators and delegators will be thinking of what will happen to current epochs rewards once the epoch ends (following week Monday). The latest section about claiming rewards is providing good amount of context related to this.